src/oss/confluent.sh

```diff
@@ -1497,9 +1497,17 @@ produce_command() {
       export_kafka
       bootstrapserver="--broker-list localhost:${kafka_port}"
     fi
-    removestr="--value-format avro"
-    if [[ "$args" =~ "$removestr" ]]; then
-      args=${args//$removestr/}
+    cloud="--cloud"
+    if [[ "$args" =~ "$cloud" ]]; then
+      file="$HOME/.ccloud/config"
+      args=${args//$cloud/--producer.config $file}
+      BOOTSTRAP_SERVERS=$( grep "^bootstrap.server" $file | awk -F'=' '{print $2;}' )
```

Comment:
awk is not very portable
https://github.com/confluentinc/confluent-cli/blob/master/src/oss/confluent.sh#L1365-L1366
you may see a two line example of how to extract a value from a key-value line where the pair is separated by `=`

Reply:
Sounds good